File: app/src/main/java/com/example/weizhenbin/wangebug/net/retrofit/entity/AliBean.java
package com.example.weizhenbin.wangebug.net.retrofit.entity;

/**
 * Created by weizhenbin on 18/8/12.
 */

public class AliBean<T> {

    public String showapi_res_error;

    public String showapi_res_id;

    public int showapi_res_code;

    public T showapi_res_body;

    @Override
    public String toString() {
        return "AliBean{" +
                "showapi_res_error='" + showapi_res_error + '\'' +
                ", showapi_res_id='" + showapi_res_id + '\'' +
                ", showapi_res_code=" + showapi_res_code +
                ", showapi_res_body=" + showapi_res_body +
                '}';
    }
}

File: app/src/main/java/com/example/weizhenbin/wangebug/net/retrofit/entity/AliBean.kt
package com.example.weizhenbin.wangebug.net.retrofit.entity

/**
 * Created by weizhenbin on 18/8/12.
 */

class AliBean<T> {

    var showapi_res_error: String? = null

    var showapi_res_id: String? = null

    var showapi_res_code: Int = 0

    var showapi_res_body: T? = null

    override fun toString(): String {
        return "AliBean{" +
                "showapi_res_error='" + showapi_res_error + '\''.toString() +
                ", showapi_res_id='" + showapi_res_id + '\''.toString() +
                ", showapi_res_code=" + showapi_res_code +
                ", showapi_res_body=" + showapi_res_body +
                '}'.toString()
    }
}
